README.arm.md

```diff
@@ -125,47 +132,32 @@ These tutorials will end up installing Ubuntu 12.04, and you have to
 upgrade to Ubuntu 14.04, or install Ubuntu 14.04 from scratch by
 finding appropriate `crouton` help.
 
-## Scaleway cloud hosted ARM servers
+#### Scaleway cloud hosted ARM servers
 
 On the current [Scaleway](http://scaleway.com) ARM servers, the Julia
 build works out of the box.
 
-## nVidia Jetson TX2
+## AArch64 (ARMv8)
 
-Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html) platform with minimal configuration changes. A full multi-threaded build, including LLVM, will complete in around two hours. All tests pass and CUDA functionality is available through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl).
+### Device specific instructions
 
-Starting from the default configuration flashed by [Jetpack 3.0](https://developer.nvidia.com/embedded/jetpack):
+#### nVidia Jetson TX2
 
-```
-sudo apt-get install libssl-dev
-```
+Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html)
+platform with minimal configuration changes. A full multi-threaded build, including LLVM,
+will complete in around two hours. All tests pass and CUDA functionality is available
+through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl).
 
-### Julia 0.5.1
-
-The easiest method to build Julia 0.5.1 is to use system provided versions of BLAS and LAPACK:
-
-```
-sudo apt-get install libopenblas-dev liblapack-dev
-```
-
-Configure Make.user as follows:
+Starting from the default configuration flashed by [Jetpack 3.0](https://developer.nvidia.com/embedded/jetpack):
 
 ```
-MARCH=armv8-a
-JULIA_CPU_TARGET=cortex-a57
-override USE_SYSTEM_BLAS=1
-override USE_SYSTEM_LAPACK=1
+sudo apt-get install libssl-dev
```

Comment:
I was actually talking about the [platform independent instructions](https://github.com/JuliaLang/julia/tree/38417ecf250bf369a5cb8c21e7b341cb9612c3e7#required-build-tools-and-external-libraries). Actually, what deps links to libssl? This is likely linux specific and not arm/aarch64 specific.

> make no distinction between requisite build software and system libraries which can be installed in order to avoid building dependencies

You mean the two lists in the section I linked above?

> refer to specific gcc versions that might not be appropriate

It mentions >= 4.7, which should be appropriate.

> don't specify libssl

It probably should in linux-specific part. Someone else with more experience building it on debian would probably know @tkelman @staticfloat